### DISCUSSION TITLE
How to Fix Meeting List Cut Off using WP Divi Theme

### DISCUSSION THREAD

**Discussion by furick1** [code4recovery/12-step-meeting-list#1376] [Q&A]
https://recoverydharma.org/meetings/?tsml-day=4

The theme or the footer and mtg list widget don't seem to work well together and info is being cut off with no ability to see additional meetings in the list. In this case you can see the full meeting address being cut off in URL above but if I add additional meetings the view won't expand the page height nor allow scrolling. I figure this is a CSS tweak due to conflict between theme and plugin but wanted to ask for help. I am using the legacy theme as the new 12 step mtg list theme breaks the view even more.

**Comment by joshreisner** [comment]
hm, pretty sure we have lots of other sites using divi without problems, i remember portland aa was using it way back, so i don't think this is specific to all divi sites.

the theme looks pretty weird with that footer floating in the middle of the screen! i notice this is true also on regular pages when they don't have enough content

the theme appears to be solving the footer overlap issue by giving the #main-content a lot of bottom margin, but our page doesn't have a `#main-content` div - you might try adding that via CSS:

```css
#tsml { 
  background-color: white;
  margin-bottom: 350px; 
}
```

also i'd suggest that background because on mobile the theme makes the page background black

<img width="1840" alt="Screenshot 2024-03-06 at 3 05 15 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/933c523d-284c-4df7-8651-37065fe4aa5a">

**Reply by furick1** [reply]
I agree the footer needs work. I just stepped in to help so will look at that further. I did add the additional css as suggested and worked. Thank you.
